python_modules/libraries/dagster-components/dagster_components/cli/list.py

```diff
@@ -56,3 +57,36 @@ def _add_component_type_to_output(
         package=package,
         **component_type.get_metadata(),
     )
+
+
+@list_cli.command(name="all-components-schema")
+@click.pass_context
+def generate_code_location_components_schema(ctx: click.Context) -> None:
+    """Builds a JSON schema which ORs the schema for a component
+    file for all component types available in the current code location.
+    """
+    assert is_inside_code_location_project(Path.cwd())
```

Comment:
nit: pls follow the style of check used in other commands (see `cli/scaffold.py`)